src/basicOp.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+/**
+ * Your task is to create a function that does four basic mathematical
+ * operations.
+ *
+ * Implement basicOp function:
+ *
+ * The function should return result of numbers after applying
+ * the chosen operation.
+ *
+ * Examples:
+ *
+ * basicOp('+', 4, 7)     // return 11
+ * basicOp('-', 15, 18)   // return -3
+ * basicOp('*', 5, 5)     // return 25
+ * basicOp('/', 49, 7)    // return 7
```

Comment:
what do we expect for non-math operations? e.g. if someone sends 'hello' as an operation?